index.js

```diff
@@ -92,6 +93,10 @@ module.exports = (md, options) => {
         md.use(require('./rules/table_wrapper'), safeObject(optsObj.table_wrapper));
     }
 
+    if (optsObj.collapsible_headings) {
```

Comment:
I remembered we discussed this would be enabled only for tutorials. So if I make this enabled by default (and transforming  all headers) we would have to turn off this plugin for every MD usage we have.

Reply:
Changed.